02week/blog/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link href="https://fonts.googleapis.com/css?family=Oswald" rel="stylesheet">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<style>
+* {
+    box-sizing: border-box;
+}
+
+</style>
+<title>Christopher Bergamo Bio</title>
+<link rel="stylesheet" href="cssblog.css">
+</head>
+<body>
+
+<div class="header">
+  <p>C. Bergamo Blog</p>
+</div>
+
+<div class="row">
+  <div class="leftcolumn">
+    <div class="card">
+      <h2>The Life of Al</h2>
+      <h5>September 1, 2018</h5>
+      <div class="blogimg" style="height:300px;"><img src="images/al.jpg"></div>
```

Comment:
make sure to set the overflow value of this container and so the picture doesn't leave its box!